hack/update-docs.sh

```diff
@@ -33,3 +36,5 @@ go run cmd/example-yaml-generator/main.go . docs
 # revert our changes
 sed -i 's/omitempty/omitgenyaml/g' pkg/crd/kubermatic/v1/*.go pkg/crd/operator/v1alpha1/*.go
 sed -i "s/,$dummy/,omitempty/g" pkg/crd/kubermatic/v1/*.go pkg/crd/operator/v1alpha1/*.go vendor/k8s.io/api/core/v1/*.go
+
+rm -rf vendor
```

Comment:
Minor: I think we could avoid removing the vendor folder as it is in `.gitignore`

Reply:
Yeah, can do.

Reply:
Fixed.